bench/DataInputOutputStreamBench.java

```diff
@@ -0,0 +1,58 @@
+package benchmark;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.DataOutputStream;
+import java.io.DataInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import com.sun.cldchi.jvm.JVM;
+
+public class DataInputOutputStreamBench {
+  void writeUTF(OutputStream out) throws IOException {
+      DataOutputStream dos = new DataOutputStream(out);
+
+      for (int i = 0; i < 25000; i++) {
+          dos.writeUTF("abcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZabcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZ");
```

Comment:
Any particular reason you left out "jk" and "wxy" from these sequences of alphabetic characters? And wouldn't it give us just as reasonable a measurement (albeit not identical) if we wrote a single sequence of a-zA-Z 50k times?